_includes/payment-url.md

```diff
@@ -59,8 +58,10 @@ you should also subscribe to other events provided by the Seamless View
 JavaScript and shut down the Seamless View if any of these events occur.
 
 Events to subscribe to for full control over the payment flow are can be found
-in {% if api_resource == "checkout" %} [Payment Menu Events](/{{ api_resource }}/other-features#payment-menu-events).
-{% else %} [Seamless View Events](/payments/{{ api_resource }}/other-features#seamless-view-events).
+in {% if api_resource == "paymentorders" %}
+{% assign url_resource= "checkout" %}[Payment Menu Events](/{{ url_resource}}/other-features#payment-menu-events).
```

Comment:
I didn't notice this URL before. We need to pass in `documentation_section` as well to this include and use that in the URL. `checkout` is going to generate wrong URLs for Payment Menu.

```suggestion
[Payment Menu Events](/{{ documentation_section }}/other-features#payment-menu-events).
```